mcs/class/corlib/corlib-net_4_x.csproj

```diff
@@ -68,6 +68,7 @@
     <Compile Include="..\..\..\external\corefx\src\Common\src\CoreLib\System\Memory.cs" />
     <Compile Include="..\..\..\external\corefx\src\Common\src\CoreLib\System\MemoryDebugView.cs" />
     <Compile Include="..\..\..\external\corefx\src\Common\src\CoreLib\System\ReadOnlyMemory.cs" />
+    <Compile Include="..\..\..\external\corefx\src\Common\src\CoreLib\System\Reflection\BindingFlags.cs" />

```

Comment:
You should do these 2 modifications to https://github.com/mono/mono/blob/master/mcs/class/corlib/corlib.dll.sources instead. This `corlib-net_4_x.csproj` is autogenerated from the `mcs/class/corlib/*.sources` files

Reply:
I have removed my changes from the generated csproj file.